Dependency Conflict with @sveltejs/kit and @sveltejs/adapter-static in sveltekit-adapter-chrome-extension

While installing. 
```
$ npm i -D sveltekit-adapter-chrome-extension

npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: sirbookmarksalot@0.0.1
npm ERR! Found: @sveltejs/kit@2.4.2
npm ERR! node_modules/@sveltejs/kit
npm ERR!   dev @sveltejs/kit@"^2.0.0" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer @sveltejs/kit@"^1.5.0" from @sveltejs/adapter-static@2.0.3
npm ERR! node_modules/@sveltejs/adapter-static
npm ERR!   peer @sveltejs/adapter-static@"^2.0.1" from sveltekit-adapter-chrome-extension@2.0.0
npm ERR!   node_modules/sveltekit-adapter-chrome-extension
npm ERR!     dev sveltekit-adapter-chrome-extension@"*" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

We need to update the versions of @sveltejs/kit, @sveltejs/adapter-static, and sveltekit-adapter-chrome-extension in your package.json file to be compatible with each other.

--- a/package.json
+++ b/package.json
@@ -27,6 +27,6 @@
     "tiny-glob": "^0.2.9"
   },
   "peerDependencies": {
-    "@sveltejs/adapter-static": "^2.0.1"
+    "@sveltejs/adapter-static": "^3.0.1"
   }
 }

